src/error.rs

```diff
@@ -29,7 +29,7 @@ impl error::Error for TarError {
 
 impl fmt::Display for TarError {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        self.desc.fmt(f)
+        write!(f, "{}: {}", self.desc, self.io)
```

Comment:
Oh for this it's intentional that the I/O error is omitted because it's hooked up in the chain of errors so if we print it here it'll get printed out in two locations

Reply:
Oh, I missed that. I wasn't seeing the error at all since my (generic) error handler only ran `fmt::Display` over them.

I'll revert this, thanks!